disinfo/api.py

```diff
@@ -8,16 +8,25 @@ def update_subreddit_names(reddit_object, search_terms):
     reddit_object : ?
     search_terms : list
     """
-    subreddits = get_subreddit_names(reddit_object, search_terms)
+
+    project_id = config["project_id"]
```

Comment:
When we move this to a cloud function, this can be accessed from env variable `GCP_PROJECT`. We could mimic this behaviour locally?